client/admin.html

```diff
@@ -215,14 +215,43 @@ <h1 class="center">Admin Panel</h1>
 							<input id="teams-enabled" type="checkbox" {{settings.teamsEnabledChecked}} />
 						</div>
 					</div>
-					<div class="row">
-						{{#each branchNames}}
+					<div class="row row-flex">
+						{{#each settings.branchRoles.applicationBranches}}
+							<div class="col-4 branch-role" data-name="{{this}}">
+								<h4>{{this}}</h4>
+								<select>
+									<option value="noop">Not used</option>
+									<option value="application" selected>Application</option>
+									<option value="confirmation">Confirmation</option>
+								</select>
+								<label>Available confirmation branches</label>
+								<fieldset>
+									{{#each ../settings.branchRoles.confirmationBranches}}
+										<div>
+											<input type="checkbox" id="{{removeSpaces ../this}}-to-{{removeSpaces this}}" data-confirmation="{{this}}" {{branchChecked ../../settings.applicationToConfirmationMap ../this this}} />
+											<label for="{{removeSpaces ../this}}-to-{{removeSpaces this}}">{{this}}</label>
+										</div>
+									{{/each}}
+								</fieldset>
+							</div>
+						{{/each}}
+						{{#each settings.branchRoles.confirmationBranches}}
+							<div class="col-4 branch-role" data-name="{{this}}">
+								<h4>{{this}}</h4>
+								<select>
+									<option value="noop">Not used</option>
+									<option value="application">Application</option>
+									<option value="confirmation" selected>Confirmation</option>
+								</select>
+							</div>
+						{{/each}}
+						{{#each settings.branchRoles.noop}}
 							<div class="col-4 branch-role" data-name="{{this}}">
 								<h4>{{this}}</h4>
 								<select>
-									<option value="noop" {{roleSelected ../settings.branchRoles "noop" this}}>Not used</option>
-									<option value="application" {{roleSelected ../settings.branchRoles "application" this}}>Application</option>
-									<option value="confirmation" {{roleSelected ../settings.branchRoles "confirmation" this}}>Confirmation</option>
+									<option value="noop" selected>Not used</option>
```

Comment:
Is `noop` the same as having all the checkboxes unchecked?

Reply:
No-op means that a question branch from `questions.json` isn't marked as an application or confirmation branch